.github/e2e_test_timeout.json

```diff
@@ -16,8 +16,8 @@
     "drainNode":                 900
   },
```

Comment:
```suggestion
    "drainNode":                 900,
    "short":                     5,
  },
```

Reply:
I think it's better to be explicit than rely on the default. As an alternative, we could use thedefault in all the "local" values.

Reply:
I'll create a ticket to rebuild this entire logic, because currently it's something with little to no sense and repeated values =(